### PR TITLE
WINC-813: Add optional Windows e2e GCP jobs

### PR DESCRIPTION
**Pull request by sebsoto**
Adds optional GCP jobs that can be triggered to run on WMCO PRs

### CODE REVIEW

File: ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml [hunk 1]
```diff
@@ -461,6 +461,83 @@ presubmits:
...
+        secret:
+          items:
+          - key: credentials
+            path: credentials
+          secretName: boskos-credentials
+      - name: ci-pull-credentials
+        secret:
+          secretName: ci-pull-credentials
+      - name: cluster-profile
+        projected:
```
Comment (saifshaikh48) [inline ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml:528]: TIL about [projected volumes](https://docs.openshift.com/container-platform/4.11/nodes/containers/nodes-containers-projected-volumes.html) :100:

**Review by saifshaikh48 — COMMENTED** [review]
/lgtm

Thanks for getting this setup @sebsoto

**Review by aravindhp — APPROVED** [review]
Thanks for working on this, @sebsoto

**Pull request merged**